script/DeployRehearsalContracts.s.sol

```diff
@@ -33,7 +33,7 @@ contract DeployRehearsalContracts is Deployer {
         require(owner_safe.isOwner(address(council_safe)));
 
         console.log("Deploying from %s", name());
-        console.log("Deployment context: %s", Config.deploymentContext());
+        console.log("Deployment context: %s", vm.envOr("DEPLOYMENT_CONTEXT", string("Deployment context not set")));
```

Comment:
Not necessarily a problem with this PR, but there are now like 4 prs that do this that are _still_ open :crying_blood: